exes/ethercat/inc/public/tfc/ec/devices/abt/easycat.hpp

```diff
@@ -56,6 +56,11 @@ class easyecat final : public base {
       last_analog_value_[i] = value;
     }
     output[0] = static_cast<std::byte>(output_states_.to_ulong() & 0x0f);
+    if (auto const servo_value = servo_.value(); servo_value.has_value()) {
+      output[1] = static_cast<std::byte>(servo_value.value());
+    } else {
+      output[1] = static_cast<std::byte>(0);
+    }
```

Comment:
this would be equivalent 
```cpp
    servo_.value().or_else([] -> std::optional<std::uint64_t> { return std::uint64_t{}; }).and_then([&output](std::uint64_t value) -> std::optional<std::uint64_t> {
      output[1] = static_cast<std::byte>(value);
      return {};
    });
```
I think the previous one is more readable in this case

Reply:
I don't think you need both an or_else and and_then. See `Returns *this if it contains a value. Otherwise, returns the result of f.`.